xdsl/ir/core.py

```diff
@@ -79,6 +84,18 @@ def split_name(name: str) -> tuple[str, str]:
         except ValueError as e:
             raise ValueError(f"Invalid operation or attribute name {name}.") from e
 
+    def get_interface(self, inter: type[DialectInterfT]) -> DialectInterfT | None:
```

Comment:
```suggestion
    def get_interface(self, interface: type[DialectInterfaceT]) -> DialectInterfaceT | None:
```

Reply:
I would personally write `interface` entirely in most places, we don't lose that much space doing so, and `inter` and `interf` are not really common shortening tbh.